_po/ja/how-to/laravel.po

```diff
@@ -586,6 +884,15 @@ msgid ""
 "                                            </dd>\n"
 "                                        </div>"
 msgstr ""
+"                                    @foreach($posts as $post)\n"
+"                                        <div class=\"border-t border-gray-200 p"
+"t-1\">\n"
+"                                            <dd class=\"sm:mt-2 text-sm text-gr"
+"ay-500\">\n"
+"                                                <span class=\"block m-1\">ID: {{"
+" $post->id }}</span>\n"
+"                                            </dd>\n"
+"                                        </div>"
```

Comment:
Probably, this is needless.
Because this test is same as before translation text.